src/helpers/daily-data-providers/health-connect-sleep.ts

```diff
@@ -0,0 +1,27 @@
+﻿import { querySleepMinutesV2 } from "./common-sleep-v2";
+
+export function totalSleepMinutes(startDate: Date, endDate: Date) {
+    return querySleepMinutesV2("HealthConnect", "sleep", startDate, endDate, [
```

Comment:
I think per @greinard we need to update the querySleepMinutesV2 to handle overlaps still...so it needs to be more like the Apple Health V1 provider.

A bit conflicted on whether we can/should ship it like this.  It would only impact people with multiple sources of sleep data - which I think would be mostly uncommon - but certanily not unheard of (Oura ring and Fitbit, or use of Apple's "InBed" timer and a Garmin, etc.)

I think we can address it in a separate PR but likely should do so soon

Reply:
I can work on a PR for that today.

Reply:
https://github.com/CareEvolution/MyDataHelpsUI/pull/441